### PR TITLE
Update admin-submission.md

### PR DESCRIPTION
**Pull request by andypunt**
Clarify the section on what happens during submission.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 1]
```diff
@@ -27,7 +27,14 @@ description: "Admins can learn how to use the Submissions portal in the Security
 
 In Microsoft 365 organizations with mailboxes in Exchange Online, admins can use the Submissions portal in the Security & Compliance Center to submit email messages, URLs, and attachments to Microsoft for scanning.
 
-When you submit an email, you will get information about any policies that may have allowed the incoming email into your tenant, as well as examination of any URLs and attachments in the mail. Policies that may have allowed a mail include an individual user's safe sender list as well as tenant level policies such as Exchange mail flow rules (also known as transport rules).
+When you submit an email, you will get:
+1) Email authentication check: details on if email authentication passed or failed when it was delivered
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
1) **Email authentication check**: Details on whether email authentication passed or failed when it was delivered.
```

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 2]
```diff
@@ -27,7 +27,14 @@ description: "Admins can learn how to use the Submissions portal in the Security
...
+2) Policy hits: information about any policies that may have allowed or blocked the incoming email into your tenant, overriding our service filter verdicts
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
2) Policy hits: information about any policies that may have allowed or blocked the incoming email into your tenant, overriding our service filter verdicts
```
```suggestion
2) **Policy hits**: Information about any policies that may have allowed or blocked the incoming email into your tenant, overriding our service filter verdicts.
```

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 3]
```diff
@@ -27,7 +27,14 @@ description: "Admins can learn how to use the Submissions portal in the Security
...
+3) Payload reputation/detonation: examination of any URLs and attachments in the message
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
3) **Payload reputation/detonation**: Examination of any URLs and attachments in the message.
```

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 4]
```diff
@@ -27,7 +27,14 @@ description: "Admins can learn how to use the Submissions portal in the Security
...
+4) Grader analysis: review done by human graders in order to confirm if messages are malicious or not
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
4) **Grader analysis**: Review done by human graders in order to confirm whether or not messages are malicious.
```

File: microsoft-365/security/office-365-security/admin-submission.md [hunk 5]
```diff
@@ -27,7 +27,14 @@ description: "Admins can learn how to use the Submissions portal in the Security
...
+
+> [!IMPORTANT]
+> Payload reputation/detonation and grader analysis is not done in all tenants. Information is blocked from going outside of the tenancy when  data is not supposed to leave the tenant boundary for compliance purposes.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/admin-submission.md]:
```suggestion
> Payload reputation/detonation and grader analysis are not done in all tenants. Information is blocked from going outside of the tenancy when data is not supposed to leave the tenant boundary for compliance purposes.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**